client/index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Schmoogle homepage</title>
+    <link rel="stylesheet" href="/client/stylesheet.css">
+    <script defer src="/client/DOM.js"></script>
+</head>
+<body>
```

Comment:
Whould have been nice if the body was clearly separated by 
header/nav bar
main body (google icon & search bar and buttons)
footer 
e.g: 
<!--nav bar-->
<!--main body-->
<!--footer-->